### ISSUE TITLE
Tab text font size

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#4656]
https://forums.zotero.org/discussion/comment/472959/#Comment_472959

Tab text size should probably increase as you increase font size?

It looks like Thunderbird does support this without increasing the tab size, and it's…fine.

![Screenshot 2024-09-02 at 4 37 24 AM](https://github.com/user-attachments/assets/0012ec43-bee6-4ed4-8d51-c0b9938fae5b)

**Comment by yexingsha** [comment]
Yes, though I think the 28px tall tabs can only comfortably display font size up to 16px:

<img width="1280" alt="library-comfortable-16px" src="https://github.com/user-attachments/assets/c12ffa4e-6b4c-4581-aef3-811fc39efe26">

20px font size would be pushing it to the extreme:

<img width="1280" alt="library-comfortable-20px" src="https://github.com/user-attachments/assets/3f27708e-01fb-423f-be97-c677f722b6e7">

But our font size goes up to 24px(!) which makes everything look ridiculous. Do we want to put some effort into polishing our UI at larger font sizes? This would involve scaling up icons (we have discussed start using 28px item type icons at certain breakpoints), buttons, search bars, and other UI elements that do not currently scale with font size.

If that's not a priority, we should still fix other places where font size do not increase properly:

1. Search bars: there was an old issue that haven't got fixed https://github.com/zotero/zotero-redesign/issues/234
2. Popups in reader: Find in Document and epub appearance settings
3. Native buttons in various dialogs
4. We should probably increase the font size in Settings as well. What about other dialogs?

**Comment by balta2ar** [comment]
I don't know if it's something with my system settings (Manjaro Linux + Openbox) or my monitor setup (2560x1440) or DPI or something, but the font in the 7.0.1 version looks painfully small :cry: 
![image](https://github.com/user-attachments/assets/0f73ef36-becb-4147-a50d-218c49fa9e4f)

Tab labels are impossible to read with my eyesight. Font size in the rest of the UI is slightly better, but if I could, I'd have increased it even more several times. Really inconvenient that there's a limit and that "Bigger" menu option gets greyed out at some point :crying_cat_face:
